backend/app/models/faq.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const FAQSchema = new Schema({
+  question: {
+    type: String,
+    trim: true,
+    required: true,
+    unique: true,
+  },
+  answer: {
+    type: String,
+    trim: true,
+    required: true,
+  },
+  isActive: {
+    type: Boolean,
+  },
+  tags: {
+    type: Array,
```

Comment:
was [String] not working?

Reply:
Arrays and [String] both of them are working